examples/gradle-kt-dsl/lib/build.gradle.kts

```diff
@@ -37,10 +37,12 @@ dependencies {
         else
             "AmazonCorrettoCryptoProvider"
 
-    if (project.hasProperty("accpLocalJar")) {
-        testImplementation(files(accpLocalJar))
-    } else {
-        testImplementation("software.amazon.cryptools:${accpArtifactId}:$accpVersion:${osdetector.classifier}")
+    if (!project.hasProperty("useBundledAccp")) {
```

Comment:
does this allow the tests to still run against the bundled JAR?

Reply:
Passing `-PuseBundledAccp` will cause the project to use ACCP that is bundled. Not specifying this option, it will either use what's on Maven or the JAR that has been built locally.